_build/config/required_externals.json

```diff
@@ -1,20 +1,26 @@
 {
-  "rex_std": {
-    "name": "rex_std",
-    "url": "https://github.com/RisingLiberty/RexStl/",
-    "tag": "v1.1.50",
-    "storage": "~/source/0_thirdparty"
-  },
-  "rex_glad": {
-    "name": "rex_glad",
-    "url": "https://gitlab.com/rex-externals/rex-glad/",
-    "tag": "v1.0.0",
-    "storage": "~/source/0_thirdparty"
-  },
-  "rex_glm": {
-    "name": "rex_glm",
-    "url": "https://gitlab.com/rex-externals/rex-glm/",
-    "tag": "v1.0.0",
-    "storage": "~/source/0_thirdparty"
-  }
+    "rex_std": {
+        "name": "rex_std",
+        "url": "https://github.com/RisingLiberty/RexStl/",
+        "tag": "v1.1.50",
+        "storage": "~/source/0_thirdparty"
+    },
+    "rex_glad": {
+        "name": "rex_glad",
+        "url": "https://gitlab.com/rex-externals/rex-glad/",
+        "tag": "v1.0.0",
+        "storage": "~/source/0_thirdparty"
+    },
+    "rex_glm": {
+        "name": "rex_glm",
+        "url": "https://gitlab.com/rex-externals/rex-glm/",
+        "tag": "v1.0.0",
+        "storage": "~/source/0_thirdparty"
+    },
+    "rex_tiny_file_dialogs": {
```

Comment:
I actually wanted to write this myself
I'm guessing it's a cross platform library to show message boxes etc?

Reply:
I don't see any added value in writing this ourselves.
Yes, it's a cross-platform implementation of a message box.

Reply:
the added value of writing it yourself is so that it can use internal rex code and be a bit more performant possibly, depends how it's implemented

Reply:
you're using this in verify which is a assert for release which is not development, so messageboxes will apear in non development builds as well.
I can also think of other cases where a messagebox will be helpful that an end user would like to see when something wrong happens

Reply:
it's non critical code, for sure and I'm sure it's not that slow, but the more it's integrated into our engine code, the better, it'll just take more time

Reply:
question is how much time we want to spend on it